.gitignore

```diff
@@ -3,6 +3,9 @@
 *.lo
 *.o
 
+# Proto files
+src/hbase/if/*.proto
```

Comment:
copying of protos can be rolled into the cmake script instead of manually invoking copy-protos script?